packages/client/components/NewMeetingCarousel.tsx

```diff
@@ -86,24 +87,49 @@ interface Props {
   idx: number
   setIdx: (idx: number) => void
   meetingOrder: MeetingTypeEnum[]
+  onStartMeetingClick: () => void
 }
 
 const NewMeetingCarousel = (props: Props) => {
-  const {idx, setIdx, meetingOrder} = props
+  const {idx, setIdx, meetingOrder, onStartMeetingClick} = props
 
   // TODO: remove when standups feature flag removed
   const moreThanThreeSlides = meetingOrder.length > 3
 
+  useHotkey('left', () => {
+    const newIdx = idx === 0 ? meetingOrder.length - 1 : idx - 1
+    setIdx(newIdx)
+  })
+  useHotkey('right', () => {
+    const newIdx = idx === meetingOrder.length - 1 ? 0 : idx + 1
+    setIdx(newIdx)
+  })
+
+  // keycode is a number but package thinks it's a string
+  const onKeyPress = (_swiper: unknown, keycode: string) => {
+    if (parseInt(keycode, 10) === 13) {
+      // inefficient, but only happens on enter
+      const isModalOpen =
+        document.querySelector(`div[id='templateModal']`) ||
+        document.querySelector(`div[id='portal']`)
+      if (!isModalOpen) {
+        onStartMeetingClick()
```

Comment:
Not sure I fully understand a comment to this piece. Why on enter `isModalOpen` becomes `false`?

Reply:
`isModalOpen` is true when a modal or portal exists in the dom. It's inefficient to search through the dom, but we only do this when the user presses the enter key, so I think it's OK. `isModalOpen` doesn't become `false` when the user presses enter.

I'd be happy to update the comment if that's unclear?